packages/common/local-storage/src/store.ts

```diff
@@ -154,6 +154,15 @@ export class LocalStorageStore<T extends object> {
     return this;
   }
 
+  /**
+   * Expunges all store-related items from local storage.
+   */
+  expunge() {
+    Object.keys(localStorage)
```

Comment:
I think this should probably iterate over the keys in the store? Because some stores share the same prefix and this might clear unexpected things

Reply:
Ah I see. I'll make that change.